pkg/metrics/machinehealthcheck.go

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright 2020 The Machine API Operator authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	DefaultHealthCheckMetricsAddress = ":8083"
```

Comment:
Why this line was removed from metrics.go and set here? https://github.com/openshift/machine-api-operator/pull/752/files#diff-4bc91a74bc47b8467cdcd25a7e3fe1651dbe44907f478377eca31b982f444f23L14

Reply:
@JoelSpeed and i had been talking about separating the metrics bits so that they could be imported in a more specific manner for each binary. as it stands now, the machinehealthcheck-controller binary will compile the entire metrics package and run the `init`. creating the`metrics/machinehealthcheck.go` is a first step in separating the different metric specific pieces. since we only use `DefaultHealthCheckMetricsAddress` in the machine health check controller, it seemed appropriate to move it here.